tests/Integration/Integrations/IntegrationsTest.php

```diff
@@ -69,8 +69,6 @@ function( $integrations ) {
 // phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
 /**
  * Class FakeIntegration2
- *
- * @package Parsely\Tests\Integrations
```

Comment:
Maybe we shouldn't remove this one

Reply:
Can you tell me what's your reasoning for this?

In the beginning I did start subcategorizing tests and it quickly became involved and too detailed (I think this is why I rolled back). If we categorize a type of test, my reasoning is that we should probably do it for all of them for consistency.

Let me know of what you think.

Reply:
Just thinking out loud, but this could also change if we restructure our tests? So we may indeed want to look at it at that time?

Reply:
That makes sense, we can leave it as is.